src/workflows/steps.js

```diff
@@ -468,3 +470,152 @@ export function setPromote( state ) {
 export function verifyConflictResolution() {
 	return git.checkConflictMarkers();
 }
+
+export function getPackageScope( state ) {
+	if ( state.qa && typeof state.qa === "boolean" ) {
+		state.qa = "@lk";
+	} else if ( state.qa && state.qa.charAt( 0 ) !== "@" ) {
+		state.qa = `@${ state.qa }`;
+	}
+
+	return Promise.resolve();
+}
+
+export function getScopedRepos( state ) {
+	const { configPath, qa: scope } = state;
+	let content = {};
+	try {
+		content = util.readJSONFile( configPath );
+
+		let repos = Object.keys( content.devDependencies ).filter( key => key.includes( scope ) );
+		repos = repos.concat( Object.keys( content.dependencies ).filter( key => key.includes( scope ) ) );
+		repos = repos.map( key => key.slice( ( scope.length + 1 ), key.length ) );
+
+		if ( repos.length === 0 ) {
+			util.advise( "noPackagesInScope" );
+			process.exit( 0 ); // eslint-disable-line no-process-exit
+		}
+
+		return Promise.resolve( repos );
+	} catch ( err ) {
+		util.advise( "updateVersion" );
+	}
+
+	return Promise.resolve();
+}
+
+export function askReposToUpdate( state ) {
+	return getScopedRepos( state ).then( packages => {
+		return util.prompt( [ {
+			type: "checkbox",
+			name: "packagesToPromote",
+			message: "Select the package(s) you wish to update:",
+			choices: packages
+		} ] ).then( ( { packagesToPromote } ) => {
+			state.packages = packagesToPromote;
```

Comment:
We could `util.advise` and `Promise.reject()` if packagesToPromote is empty... or did I miss that somewhere else?